tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py27,py35}-django{111},py35-django{20,21,22}
+envlist = {py27,py35}-django{111},py35-django{20,21,22},py38-django{22}
```

Comment:
Shouldn't py27 be removed from here as well?

Reply:
good catch.